docs/concepts.md

```diff
@@ -33,6 +33,24 @@ When there is late arriving data (data intended for 9:00 arriving >1 hr late at
 With the help of the timeline, an incremental query attempting to get all new data that was committed successfully since 10:00 hours, is able to very efficiently consume
 only the changed files without say scanning all the time buckets > 07:00.
 
+## Terminologies
+
+ * `Hudi Dataset` 
+    A structured hive/spark table managed by Hudi. Hudi supports both partitioned and non-partitioned Hive tables. 
+ * `Commit` 
```

Comment:
COMMITS/CLEANS/COMPACTIONS are already explained above this?

Reply:
Should we add DELTA_COMMITS as well there?

Reply:
yes, done